applications/GeoMechanicsApplication/custom_conditions/Pw_condition.cpp

```diff
@@ -43,37 +42,35 @@ void PwCondition<TDim, TNumNodes>::GetDofList(DofsVectorType& rConditionDofList,
 }
 
 //----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
Yes, I totally agree. However, this will be addressed separately. See issue https://github.com/orgs/Deltares/projects/55/views/8?pane=issue&itemId=66505846.